catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -241,7 +240,7 @@ private QueryResponse sourceFederate(List<Source> sources, final QueryRequest qu
 
                     QueryRequest sourceQueryRequest = new QueryRequestImpl(modifiedQuery,
                             queryRequest.isEnterprise(),
-                            Collections.singleton(source.getId()),
+                            queryRequest.getSourceIds(),
```

Comment:
I don't think this should be an issue but I will create the new hash set to be on the safe side.